### PR TITLE
Adding booking list

### PR DESCRIPTION
**Pull request by carlos-sfl**
Added retrieving of bookings list.

### CODE REVIEW

**Comment by jan-carreras** [comment]
This library supports versions prior to PHP 7.x, thus return type syntax is not supported. It needs to be addressed.

File: src/Application/Hotel/Exception/BookingInfoListException.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace StayForLong\Juniper\Application\Hotel\Exception;
+
+
+final class BookingInfoListException extends \Exception
+{
+	public static function throwBecauseOf($messages)
```
Comment (jan-carreras) [inline src/Application/Hotel/Exception/BookingInfoListException.php]: What about creating a
```public static function becauseOf($message) {
return new self($message);
}```

and make the throw itself on the callers side?

File: src/Domain/Hotel/BookingInfo.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	 */
+	public function BookingVoucher(): string
+	{
+		return $this->bookingVoucher;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function BookingStatus(): string
```
Comment (jan-carreras) [inline src/Domain/Hotel/BookingInfo.php]: Watch out! Getters must be in 'lower camel case' style `bookingStatus()`

Comment (carlos-sfl) [inline src/Domain/Hotel/BookingInfo.php]: the IDE code generator adds uppercase... I change it!

**Pull request merged**